datalad/distributed/create_sibling_ria.py

```diff
@@ -679,8 +680,7 @@ def _create_sibling_ria(
         if post_update_hook:
             disabled_hook.rename(enabled_hook)
         if group:
-            # TODO; do we need a cwd here?
-            subprocess.run(chgrp_cmd, cwd=ds.path)
+            Runner().run(chgrp_cmd)
```

Comment:
The answer is no, since all paths handled in that call are absolute. However, instead of the removed comment I may be should add one, pointing out, that it isn't needed ;-)